.github/workflows/update-third-party-notices.yml

```diff
@@ -0,0 +1,27 @@
+name: Check and Update Third Party Notices
+on: workflow_call
+
+jobs:
+  license-check:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: "https://registry.npmjs.org"
+      - run: npm install -g generate-license-file@1
```

Comment:
I would add this to our package.json. It's not some huge package right?

Reply:
Then it'll also benefit from our caching action. Technically we can still set up separate caching just for this but it seems unnecessary

Reply:
nup I can do that!